podjs/src/browserPod.ts

```diff
@@ -159,7 +159,41 @@ class LocalStoragePolyOut implements PolyOut {
     }
 
     stat(id: string): Promise<Stats> {
-        return new Promise((resolve) => {
+        return new Promise((resolve, reject) => {
```

Comment:
OK, I see. As far as I understand this, and please @fhd correct me if I'm wrong, this is not what's actually used in production; browserPod is mainly for testing purposes. The real deal is remotepod. So If you're using this just for testing, in principle it's OK, but this is not going to propagate into production unless you deal with that. Even so, I really love to keep separated things separated. This implies a change that's outside what looks like the scope of this PR, which is mainly in the feature realm. So in that spirit, and if this is really necessary, I would really like to see this in a separate PR, with tests and maybe a bit of discussion about the API and whether that API should propagate all the way down to the pod, or to other mocks like DefaultPod (which will become MockPod shortly)

Reply:
I think the problem is that `stat` was previously not implemented in podjs for files _inside_ a ZIP file. Apparently that got us this far, but to test his changes locally, Andrei had to add an implementation of it.

Given how `BrowserPod` isn't exactly production ready at this point and has zero tests right now, adding what we need _when_ we need it is fine by me. There is an open PR (that I need to pick back up, it's been rotting...) about getting podjs a _bit_ more production ready. But right now it's primarily a tool for faster development / manual testing.